ui/lib/config-ui/addon/components/messages/page/details.js

```diff
@@ -6,16 +6,17 @@
 import Component from '@glimmer/component';
 import { service } from '@ember/service';
 import { action } from '@ember/object';
-import errorMessage from 'vault/utils/error-message';
+import apiErrorMessage from 'vault/utils/api-error-message';
 
 /**
  * @module Page::MessageDetails
  * Page::MessageDetails components are used to display a message
  * @example
  * ```js
- * <Page::MessageDetails @message={{this.message}}  />
+ * <Page::MessageDetails @message={{this.model.message}} @capabilities={{this.model.capabilities}}  />
  * ```
- * @param {model} message - message model
+ * @param message - message model
```

Comment:
Are we still referring to these resources as "models"? Doesn't really matter either way, just wondering if we should call it something else like "message resource" or "message data object"

Reply:
No I'll go ahead and change that comment. I'm not sure more explanation is actually needed here do you? I think the arg name implies what it is. Do you think it's sufficient to update it to `@param message`?

Reply:
Yeah, that definitely seems sufficient 😄